EventHubs/src/README.md

```diff
@@ -1,39 +1,32 @@
 
-## Setup for Gcm.Samples.LogicApp
+You download an ARM template, edit it to add the URL of your HTTP source, copy the template into Azure Portal, and deploy it. Then, you can start exporting monitoring data to EventHub.
 
-- Replace the content of the file AzureLogicApp.json with your Logic App code (extracted from Visual Studio or Azure Portal).
-- Add the default parameters: 
-  - applicationName: provided by default, using the value of the **Application** setting from [deploy.app.json](Config/deploy.app.json).
-  - lifecycle: provided by default, using the value of the **Lifecycle** setting from [deploy.app.json](Config/deploy.app.json).
-  - lifecycleSuffix: provided by default, using the value of the **LifecycleSuffix** setting, from [deploy.app.json](Config/deploy.app.json).
-- Clean up your ARM Template to avoid hardcoded values e.g. subscription key.
-- Fill in 'Config/deploy.app.json' and 'Config/deploy.variables.json' with your Logic App's configuration. Every parameter from the ARM Template that requires a value (besides the default parameters listed above) have to be added as **TemplateParameters** with an object notation, not as an array.
+This solution enables you to collect:
 
+* [Activity Logs](https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/Azure-Audit/02Collect-Logs-for-Azure-Audit-from-Event-Hub)
+* [Diagnostics Logs](https://help.sumologic.com/Send-Data/Collect-from-Other-Data-Sources/Azure_Monitoring/Collect_Logs_from_Azure_Monitor) and [Metrics](https://help.sumologic.com/Send-Data/Collect-from-Other-Data-Sources/Azure_Monitoring/Collect_Metrics_from_Azure_Monitor) which can be exported via Azure Monitor
 
-## Deploy Parameters
-> Bold parameters are required.
-- **TemplateName** (string): The name of the template without file extension to be used during deployment. Templates should be a json file in the root directory. By default it's named AzureLogicApp, so if you change the ARM Template file name, also change the **TemplateName** variable.
+![EventHub Collection Data Pipeline](https://s3.amazonaws.com/appdev-cloudformation-templates/AzureEventHubCollection.png)
 
-- **ApplicationPrefix** (string): Used as second part of the full application name. Possible values: pub, pvm, ent. If ent(enterprise), this will be null.
+## Building the function
+Currently ARM template is integrated with github and for each functions
+EventHubs/target/logs_build/EventHubs_Logs - Function for ingesting Activity Logs
+EventHubs/target/metrics_build/EventHubs_Metrics - Function for ingesting Metrics Data
 
-- **Application** (string): The name of the Logic App. This will provide a value to the **applicationName** default parameter from the ARM Template. Should not include dashes (-).
+## For Developers
+`npm run build`
+This command copies required files in two directories logs_build(used for activity logs ingestions) and metrics_build(used for metrics data(in diagnostic settings) ingestion)
```

Comment:
Have you used this section - for the tests etc. If not, we can probably remove it and the associated python tests